src/webpack.config.js

```diff
@@ -48,76 +56,94 @@ module.exports = function (args) {
 		}));
 	}
 
-    const webpackConfig = {
-        externals: [
-            function (context, request, callback) {
-                if (/^intern[!\/]/.test(request)) {
-                    return callback(null, "amd " + request);
-                }
-                callback();
-            }
-        ],
-        entry: {
-            'src/main': [
-                path.join(basePath, 'src/main.styl'),
-                path.join(basePath, 'src/main.ts')
-            ]
-        },
-        plugins: plugins,
-        output: {
-            libraryTarget: 'umd',
-            path: path.resolve('./dist'),
-            filename: '[name].js'
-        },
-        devtool: 'source-map',
-        resolve: {
-            root: [ basePath ],
-            extensions: ['', '.ts', '.tsx', '.js'],
-            alias: {
-                rxjs: '@reactivex/rxjs/dist/amd'
-            }
-        },
-        resolveLoader: {
-            root: [ path.join(__dirname, 'node_modules') ]
-        },
-        module: {
-            preLoaders: [
-                {
-                    test: /dojo-.*\.js$/,
-                    loader: 'source-map-loader'
-                }
-            ],
-            loaders: [
-                { test: /src[\\\/].*\.ts?$/, loader: 'umd-compat-loader!ts-loader' },
-                { test: /\.js?$/, loader: 'umd-compat-loader' },
+	const webpackConfig = {
+		externals: [
+			function (context, request, callback) {
+				if (/^intern[!\/]/.test(request)) {
+					return callback(null, "amd " + request);
+				}
+				callback();
+			}
+		],
+		entry: {
+			'src/main': [
+				path.join(basePath, 'src/main.css'),
+				path.join(basePath, 'src/main.ts')
+			]
+		},
+		plugins: plugins,
+		postcss: [
+			postcssImport,
+			postcssCssNext({
+				features: {
+					autoprefixer: {
+						browsers: [
+							'last 2 versions',
+							'ie >= 10'
+						]
+					}
+				}
+			})
+		],
+		output: {
+			libraryTarget: 'umd',
+			path: path.resolve('./dist'),
+			filename: '[name].js'
+		},
+		devtool: 'source-map',
+		resolve: {
+			root: [ basePath ],
+			extensions: ['', '.ts', '.tsx', '.js', '.css.json'],
```

Comment:
not sure why it's there, but again probably not a change that's in the scope of this PR